.eslintrc.yml

```diff
@@ -52,6 +54,22 @@ rules:
   react/jsx-props-no-spreading: "off"
   react/button-has-type: "off"
   "@typescript-eslint/type-annotation-spacing": error
+  check-file/filename-naming-convention:
+    - error
+    - "**/*.{jsx,tsx,ts,js}": "KEBAB_CASE"
```

Comment:
Let's add `css`, `scss` and `sass` files here. Also maybe `svg`, `jpg`, `jpeg` and `png`.